storage/tests/unit/test_blob.py

```diff
@@ -366,7 +366,7 @@ def test__make_transport(self, fake_session_factory):
 
     def test__get_download_url_with_media_link(self):
         blob_name = 'something.txt'
-        bucket = mock.Mock(spec=[])
+        bucket = _Bucket(name='IRRELEVANT')
```

Comment:
OK.  Please note that the "mock everything explicitly in each test" pattern makes for a lot of churn:  rather than adding a single line to the (already existing) `_Bucket` class, I have to add one or more lines to basically every test in the module which touches an API call (I switched them to re-using the existing `_Bucket` class in #3495).